.github/workflows/push.yml

```diff
@@ -7,15 +7,15 @@ jobs:
     name: Test pushes
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@master
-      if: github['ref'] == 'refs/heads/master'
-    - name: Use Node.js 10.x
-      if: github['ref'] == 'refs/heads/master'
-      uses: actions/setup-node@v1
-      with:
-        node-version: 10.x
-    - name: npm install, build, and test
-      if: github['ref'] == 'refs/heads/master'
-      run: |
-        npm ci
-        npm test
+      - uses: actions/checkout@master
+        if: github['ref'] == 'refs/heads/master'
+      - name: Use Node.js 10.x
+        if: github['ref'] == 'refs/heads/master'
+        uses: actions/setup-node@v1
+        with:
+          node-version: 10.x
+      - name: npm install, build, and test
+        if: github['ref'] == 'refs/heads/master'
```

Comment:
Again, maybe for another PR, but shouldn't the branch check be on the `on: push` instead of on each step?

```yaml
on:
  push:
    branches:
      - master
```